src/jdk.localedata/share/classes/sun/util/resources/ext/CurrencyNames_de.properties

```diff
@@ -60,218 +60,450 @@
 # or other dealings in these Data Files or Software without prior
 # written authorization of the copyright holder.
 
-#
-# Generated automatically from the Common Locale Data Repository. DO NOT EDIT!
-#
+ADP=ADP
```

Comment:
No need for these lines in each localized files, as English bundle contains them.